UIforETW/UIforETWDlg.cpp

```diff
@@ -282,7 +280,7 @@ BOOL CUIforETWDlg::OnInitDialog()
 		}
 	}
 
-	if (GetWindowsVersion() == kWindowsVersionXP)
+	if (!IsWindowsVistaOrGreater())

```

Comment:
[Your comment below](https://github.com/google/UIforETW/pull/56/files#r41416669) is a case in point.